### DISCUSSION TITLE
Problems with data entry

### DISCUSSION THREAD

**Discussion by SDGArmenia** [open-sdg/open-sdg#2149] [Q&A]
Hi Everyone,

Can anybody help solve the problem?

After updating the SDG version, there was a problem with  data entry.
For new data entry or editing an indicator, we do not have a Save and Edit button.

To enter data , we do the following steps in https://armstat.github.io/sdg-site-armenia/1-1-1/data/

1. enter data, download using the Download button,
2. open the downloaded file using the Notepad application and delete the **empty line** created after entering the data.
![image](https://github.com/user-attachments/assets/f6ea411b-dfc6-4c3b-9973-a34525e44c20)

3. in the application, click Go to repository and upload the file.

Since the ministries of the RA also must to enter their data, can we
1. create "Save" and "Edit" buttons to close access to the "Go to repository" button for them?
2. How to solve the issue of an empty line that is created with each change or can it possible to have Add button to avoid  last empty line?

Thanks in advance

**Comment by brockfanning** [comment]
@SDGArmenia Thank you for pointing out this issue with the empty line. We'll work on a fix for that.

Regarding the "Save" button. This is something that is possible but requires some additional setup and the usage of third-party services. We are also working on drafting a tutorial for this.

**Reply by SDGArmenia** [reply]
Dear @brockfanning, We have a workshop at the end of next week, is there anything we can do before then?
